src/main/java/uk/gov/hmcts/befta/util/DynamicValueInjector.java

```diff
@@ -120,12 +121,15 @@ else if (anEnvVarIsStartingAt(input, pos)) {
             }
             if (jumpTo > 0) {
                 pos = jumpTo;
+                if (partValue instanceof Long || partValue instanceof Integer){
```

Comment:
This would be better:
`partValue instanceof Number`

Reply:
done